### PR TITLE
fix pump server_test

### PR DESCRIPTION
**Pull request by lichunzhu**
### What problem does this PR solve? 
Refine Pump server test to avoid fault in `TestStartPumpServer`.

### What is changed and how it works?
Change `string` to `strconv.FormatInt`.

### Check List 

Tests 

 - Unit test
 - Integration test

Code changes

Side effects

Related changes

### CODE REVIEW

**Comment by lichunzhu** [comment]
/run-all-tests tidb=release-3.0 tikv=release-3.0 pd=release-3.0

File: pump/server_test.go [hunk 1]
```diff
@@ -709,7 +710,7 @@ func (s *startServerSuite) TestStartPumpServer(c *C) {
 	cfg := &Config{
 		ListenAddr:        "http://127.0.0.1:8250",
 		AdvertiseAddr:     "http://127.0.0.1:8260",
-		Socket:            "unix://127.0.0.1:" + string(time.Now().UnixNano()) + "/hello/world",
+		Socket:            "unix://127.0.0.1:" + strconv.FormatInt(time.Now().UnixNano(), 10) + "/hello/world",
```
Comment (july2993) [inline pump/server_test.go]: there's no way a port can bigger than 65536, but this code works, look strange, maybe better change it.

**Comment by lichunzhu** [comment]
/run-all-tests tidb=release-3.0 tikv=release-3.0 pd=release-3.0

**Pull request merged**